lib/tasks/bower-install.js

```diff
@@ -3,68 +3,97 @@
 // Runs `bower install` in cwd
 var path       = require('path');
 var existsSync = require('exists-sync');
+var execa = require('execa');
 var Promise    = require('../ext/promise');
 var Task       = require('../models/task');
 var formatPackageList = require('../utilities/format-package-list');
+var resolve = Promise.denodeify(require('resolve'));
 
 module.exports = Task.extend({
-  init: function() {
-    this._super.apply(this, arguments);
-    this.bower = this.bower || require('bower');
-    this.bowerConfig = this.bowerConfig || require('bower-config');
+
+  ensureBower: function() {
+    if (this.bower) {
+      return Promise.resolve(this.bower);
+    }
+
+    return resolve('bower').catch(function() {
+      return this.installBower().then(function() {
+        return resolve('bower');
+      });
+    }.bind(this)).then(function(bowerPath) {
+      return require(bowerPath);
+    });
+  },
+
+  installBower: function() {
+    var ui = this.ui;
+    var chalk = require('chalk');
+
+    ui.startProgress(chalk.green('NPM: Installing bower ...'));
+
+    return Promise.cast(execa('npm', ['install', 'bower@^1.3.12'], {
```

Comment:
Haha, well we can fix this one usage here (since we have to change it anyways) and then go on a hunt...